test/check-machines-settings

```diff
@@ -253,8 +253,7 @@ class TestMachinesSettings(VirtualMachinesCase):
         b = self.browser
 
         self.createVm("subVmTest1")
-        self.machine.execute("qemu-img create -f raw /var/lib/libvirt/images/phonycdrom 1M; "
-                             "virsh attach-disk subVmTest1 /var/lib/libvirt/images/phonycdrom sdb --type cdrom --config")
+        self.machine.execute("touch /var/lib/libvirt/images/phonycdrom; virsh attach-disk subVmTest1 /var/lib/libvirt/images/phonycdrom sdb --type cdrom --config")
```

Comment:
The two-line version was easier to read/review.

Reply:
I just reverted the commit, but I can also partially revert it. and keep this and the comment from above.